python/tank/api.py

```diff
@@ -609,6 +609,28 @@ def tank_from_entity(entity_type, entity_id):
     pc = pipelineconfig_factory.from_entity(entity_type, entity_id)
     return Tank(pc)
 
+
+_current_user = None
+
+
+def set_current_user(user):
+    """
+    Sets the current Shotgun user.
+    :params user: a shotgun_authentication.ShotgunUser derived object. Can be None to clear the
+                  current user.
+    """
+    global _current_user
+    _current_user = user
+
+
+def get_current_user():
+    """
+    Returns the current Shotgun user.
```

Comment:
Do we need this? It seems to be used in one place, where there are some other CR comments and suggestions. It is also a little confusing that there is a `tank.get_current_user()` and a `tank.util.get_current_user()`